content-api/content-service/conf/logback.xml

```diff
@@ -11,8 +11,22 @@
 		</encoder>
 	</appender>
 
+	<appender name="rollingFile" class="ch.qos.logback.core.rolling.RollingFileAppender">
+		<rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+			<fileNamePattern>/data/logs/content-log-%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
```suggestion
			<fileNamePattern>/data/logs/search-log-%d{yyyy-MM-dd}.log</fileNamePattern>
```

Reply:
This change was in content-api, updated the logback accordingly